installer.sh

```diff
@@ -360,37 +360,48 @@ else
   CHANGEPASSWORD="y"
 fi
 
-if [ "$CHANGEPASSWORD" == "y" ]; then
-  unset CHARCOUNT
-  echo -n "Set the password to access the Dashboard: "
-  CHARCOUNT=0
+
+read_password() {
+  local CHARCOUNT=0
+  local PASSWORD=""
   while IFS= read -p "$PROMPT" -r -s -n 1 CHAR
   do
     # Enter - accept password
     if [[ $CHAR == $'\0' ]] ; then
```

Comment:
this might be worth looking into. chatGPT keeps harping that \0 isnt the end of a string in a shell script when a user presses ENTER, and that we should be using \n instead

Reply:
this part was only moved no code change

Reply:
\n don't work